### DISCUSSION TITLE
Bugs with Lightdm and Arch Linux with GNOME Desktop

### DISCUSSION THREAD

**Discussion by mrp4sten** [1amSimp1e/dots#35] [Q&A]
I have some errors, I've already read and follow the steps for this awesome **Hyprland** Set-Up but I'm using **Lightdm** like Login Manager and my OS base is **ArchLinux** with **GNOME** Desktop like main Window Manager, but after follow the Steps for Late Night theme.

Im Select the Window Manager Hyprland and after do login, automatically Im back in the Lightdm, Im change Lightm to GDM and After login to Hyprland manager works, but Some error appears in the Top.

- Problems in hyperlnd file config at the line **119**
- Problems with **xdg-desktop-portal**.

I dont know whats the problem Exactly can you help me to resolve this?

**Comment by 1amSimp1e** [comment]
actually you can't login to hyprland using `lightdm` as said in the [wiki](https://wiki.hyprland.org/Getting-Started/Master-Tutorial/#launching-hyprland). Second problems at 119 as you said, if you want to use [vfr](https://wiki.hyprland.org/Configuring/Variables/#misc) enable it using this `vfr = true` or disable `vfr = false`. Also, the problems with `xdg-desktop-portal` read [this](https://wiki.hyprland.org/Useful-Utilities/Hyprland-desktop-portal/)

**Reply by mrp4sten** [reply]
Thank U for the information Dude and I'm so sorry to take your time, For the next time I will read the Documentation. 
Thanks @1amSimp1e :blush:

**Comment by 1amSimp1e** [comment]
your welcome
